src/numbers/index.js

```diff
@@ -31,7 +28,7 @@ export { XdrLargeInt };
  * @throws {TypeError} if the `scv` input value doesn't represent an integer
  */
 export function scValToBigInt(scv) {
-  const type = scv.switch().name.slice(3).toLowerCase();
+  const type = scv.switch().name.slice(3).toLowerCase(); // if it ain't broke...
```

Comment:
can scval.js encapsulate how to obtain scval type as `getType(xdr.ScVal)`, the `slice(3)` part seems really low level aspect which this conversion routine probably doesn't need to know about?

Reply:
I disagree that this needs to be pulled out since it isn't being reused anywhere, but I do see what you're getting at from a self-documenting code perspective. Done in 06b9adec, take a look!

Reply:
looks good!